common/src/main/res/values/colors.xml

```diff
@@ -115,4 +115,7 @@
     <color name="gray_200">#C4C8CC</color>
     <color name="gray_300">#AAAEB3</color>
     <color name="gray_600">#5D5F61</color>
+
+    <color name="dash_blue_40">#0A008DE4</color>
+    <color name="dark_text">#3c3c3c</color>
```

Comment:
These are legacy color values from a few months ago.  Need to find their replacements.

Reply:
`dark_text` -> `content_primary`
`dash_blue_40` can be kept since it's a `dash_blue` with alpha. But perhaps rename it to `dash_blue_0.04` since it's 4% alpha. Or just use `dash_blue_0.05` which is basically the same.